docs/docs/reference/crd-reference/taskdefinition.md

```diff
@@ -32,27 +32,30 @@ differentiated by the `spec` section:
   [Synopsis for container-runtime](#synopsis-for-container-runtime)
   and
   [Examples for a container-runtime runner](#examples-for-a-container-runtime-runner).
-- Pre-defined containers
-
-    - Use the pre-defined `deno-runtime` runner
-      to define tasks using
-      [Deno](https://deno.com/)
-      scripts,
-      which use a syntax similar to JavaScript and Typescript,
-      with a few limitations.
-      You can use this to specify simple actions
-      without having to define a full container.
-      See
-      [Synopsis for Deno-runtime container](#deno-runtime)
-      and
-      [Deno-runtime examples](#examples-for-deno-runtime-runner).
-    - Use the pre-defined `python-runtime` runner
-      to define your task using
-      [Python 3](https://www.python.org/).
-      See
-      [Synopsis for python-runtime runner](#python-runtime)
-      and
-      [Examples for a python-runtime runner](#examples-for-a-python-runtime-runner).
+
+<!-- markdownlint-disable MD046 -->
```

Comment:
Let's not disable the linter for the whole page, but only for the part, where tabs are used (lines aprox. 380-485). The rest of the page should be ok

Reply:
Done.
Now disabling the MD046 linter for the required sections [L156 - L182](https://github.com/keptn/lifecycle-toolkit/pull/3005/commits/7ae837c1b73aa6d078c120d3c55161fa70074625#diff-85b5836a0cfd1c2c23657e6e340dd43f2eeb53b8bebb5c65246cf10ef8a22db8R152-R183) and [L363 - L467](https://github.com/keptn/lifecycle-toolkit/pull/3005/commits/7ae837c1b73aa6d078c120d3c55161fa70074625#diff-85b5836a0cfd1c2c23657e6e340dd43f2eeb53b8bebb5c65246cf10ef8a22db8R363-R467) only.